web/customize_viewer.css

```diff
@@ -2,31 +2,160 @@
  * Custom css for our fork of pdf.js.
  */
 
+html body.light {
```

Comment:
Yes. This is overriding rules like this:

```css
html[dir='ltr'] body {}
```

It's not necessary for every selector, but I thought it was most readable if I used it consistently.